--- auto-multiple-choice-1.5.1-original/auto-multiple-choice.in	2021-10-25 15:45:02.289497839 +0200
+++ auto-multiple-choice-1.5.1/auto-multiple-choice.in	2021-10-25 15:47:57.944367515 +0200
@@ -48,7 +48,7 @@
     chdir($base_dir);
     $base_dir=getcwd();
     chdir($wd);
-    $mods_dir=catfile($base_dir,'lib');
+    $mods_dir=catfile($base_dir,'lib', 'AMC');
     if(! -d $ENV{'AMCBASEDIR'}) {
 	$ENV{'PERL5LIB'}=catfile($base_dir,'perl').":".$ENV{'PERL5LIB'};
 	$ENV{'TEXINPUTS'}=catfile($base_dir,'latex').":".$ENV{'TEXINPUTS'};
